accession/utils.py

```diff
@@ -0,0 +1,123 @@
+"""Returns configurations for exporting querysets to csv based on the model.
```

Comment:
This should be changed to something more generic, since modules don't actual return anything